src/assets/img/icons/create-staking-icon.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Would be awesome if you minify this svg for example with https://jakearchibald.github.io/svgomg/ :)